app/views/users/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1>List of users</h1>
\ No newline at end of file
+<h1>List of users</h1>
+<% @users.reverse_each do |user| %>
+  <%= link_to user do %>
+    <%= render user %> 
+  <% end %>
+<% end %>
```

Comment:
Nicely done! Users controller `#index` action renders all users to it's corresponding Views index page (`app/views/users/index.html.erb`) 😎